apps/web/src/components/Composer/NewPublication.tsx

```diff
@@ -547,7 +548,7 @@ const NewPublication: FC<NewPublicationProps> = ({ publication }) => {
           <Attachment />
           <Giphy setGifAttachment={(gif: IGif) => setGifAttachment(gif)} />
           <CollectSettings />
-          <ReferenceSettings />
+          <NewReferenceSettings />
```

Comment:
Replace old `ReferenceSettings` to new one, don't want to give a new name called `NewReferenceSettings`

Reply:
Will do